front/src/components/table/table-header/SortAndFilterBar.tsx

```diff
@@ -24,6 +24,26 @@ const StyledBar = styled.div`
   height: 40px;
 `;
 
+const StyledFilterButton = styled.button`
+  margin-left: auto;
```

Comment:
I'm using flex usually for alignment but I'm fine with margin-left: auto too

Reply:
This was made in the spirit of this blog post specifically about aligning the last item in a flex:

https://medium.com/@iamryanyu/how-to-align-last-flex-item-to-right-73512e4e5912